core/sail/federation/src/main/java/org/eclipse/rdf4j/sail/federation/Federation.java

```diff
@@ -94,6 +100,18 @@ public void addMember(Repository member) {
 		members.add(member);
 	}
 
+	protected List<Repository> getMembers() {
+		return Collections.unmodifiableList(members);
+	}
+
+	public void setBloomFilter(Repository member, RepositoryBloomFilter filter) {
```

Comment:
This needs javadoc to explain its assumptions, in particular, that not every Repository in getMembers needs to have this method called on it if the user doesn't need to override the default implementation.